internal/pkg/service/buffer/worker/service/slice_retry_test.go

```diff
@@ -251,12 +251,11 @@ slice/active/closed/failed/00000123/my-receiver-1/my-export-1/0001-01-01T00:00:0
 >>>>>
 
 <<<<<
-task/00000123/my-receiver-1/my-export-1/slice.close/%s
+task/00000123/my-receiver-1/slice.close/%s
```

Comment:
No a tady se nějak pořád dostává export id do zámku a netuším jak:
![CleanShot 2023-02-08 at 23 09 25@2x](https://user-images.githubusercontent.com/215660/217662792-ad05284c-69fe-433d-a93e-602586fb049b.jpg)

Reply:
Mno, tak už vím. Je to odtud: https://github.com/keboola/keboola-as-code/blob/1054370b2bf068ae165a69da68c6ad5ff5fda303/internal/pkg/service/buffer/worker/task/orchestrator/orchestrator.go#L132-L135

![image](https://user-images.githubusercontent.com/215660/217811776-d156cdbf-9320-4888-9c34-f651171b7ca0.png)

Tím, že se odstranil export id z toho key, tak `TrimPrefix()` ho neodstraní z `value` kde je zřejmě slice key. Tím pádem zůstane export id v lock key. Je to ok? 🤔

Reply:
Tu je potom druha strana skladacky: 
https://github.com/keboola/keboola-as-code/blob/1054370b2bf068ae165a69da68c6ad5ff5fda303/internal/pkg/service/buffer/worker/task/node.go#L129

Reply:
Mozno by bolo najlepsie, ulozit cely `lock`:

Ukladanie:
```
lock := w.config.TaskType + "/" + resourceID (value.String())
```
Pouzitie:
```
n.schema.Runtime().Lock().Task().(task.Lock) 
```

--------------


Tym by sa mohol znizit "confusing factor", co myslis?

Bolo by potrebne upravit par testov, ale nic hrozne.

Reply:
No nevím. Zkusil jsem: https://github.com/keboola/keboola-as-code/pull/1183/commits/1a44e9432affc1c058c2e11fd892149009688042 ale zas jsem zasekanej na třech padajících testech se kterýma nemůžu hnout. 😩 Já to tam prostě nevidím.

![CleanShot 2023-02-09 at 14 22 07](https://user-images.githubusercontent.com/215660/217824392-d5b45fad-fd84-4fed-9e62-415451836665.jpg)

![CleanShot 2023-02-09 at 14 23 03](https://user-images.githubusercontent.com/215660/217824559-7eab6812-0ae2-4c53-9a50-24c09116ec8b.jpg)

![CleanShot 2023-02-09 at 14 23 20](https://user-images.githubusercontent.com/215660/217824612-fe7ef3a3-1c55-42d7-a007-37b310509235.jpg)

Reply:
Zachvilku to pozriem.

Reply:
:eyes:

Reply:
Ked neviem najst rozdiel, tak si to nakopirujem do VsCode a spravim diff.

Zatial som nasiel jeden preklep, idem pozriet ostatne.

![image](https://user-images.githubusercontent.com/19371734/217845246-51c13f3d-4b3e-481f-a0fc-a3151f8a74c8.png)

Reply:
Boli tam len drobne preklepy: https://github.com/keboola/keboola-as-code/pull/1183/commits/0bc63baf7f5d2f055beb568f0afcf65cb22ea014 :crossed_fingers:

Reply:
(este som to nepozeral cele, iba fixol tie testy)

Reply:
Ach jo, ok, diff, děkuju. 🙂 Ve VSCode jsem to měl, ale zkoumal jsem kde mám špatně nějakou mezeru nebo tabulátor.